api/v3/Case/Getdetails.php

```diff
@@ -154,6 +154,7 @@ function civicrm_api3_case_getdetails(array $params) {
         'case_id' => ['IN' => $ids],
         'is_current_revision' => 1,
         'is_test' => 0,
+        'activity_type_id' => ['!=' => "Bulk Email"],
```

Comment:
Bulk email can be in single quotes

Reply:
Hope there are no other places in the custom API's where Activity.get is being called and we need to exclude the bulk email activity?

Reply:
Good call. Removed the Bulk Email from one more place.